### ISSUE TITLE
Incorrect drawing of the nested table

### ISSUE THREAD

**Issue by LBorowyk** [bpampuch/pdfmake#2069]
I am trying to draw a nested table in merged cells. But the contents of cell 11 are displayed below the contents of cell 00.
```js
var dd = {
    content: [
        {
            table: {
                body:[
                    [
                        {
                            rowSpan: 2,
                            table: {
                                body: [
                                    [
                                        '000', '001'
                                    ]
                                ]
                            }
                        },
                        { text: '01' },

                    ],
                    [
                        { text: '10' },
                        { text: '11' },
                    ],
                ]
            }
        }
    ],
}
```

**Comment by suchislife801** [comment]
Why are you nesting tables? What are you, in the 90's? Stop it. Stop it now.

**Comment by mjomble** [comment]
Found and marked some duplicates of this issue. Adding an image for clarity:

![image](https://user-images.githubusercontent.com/2202751/193459383-87332dc1-54ba-448e-90c0-c8e7f0a28e0c.png)

I may be able to look into fixing this, but no promises 🙂

PS. Another example where the same issue occurs with columns inside table cell:

```
var dd = {
	content: {
        table: {
            body: [
                [
                    { rowSpan: 2, columns: ['This is ok'] },
                    'This is ok',
                ],
                ['', 'This should be on the right'],
            ],
        },
    }
}
```

![image](https://user-images.githubusercontent.com/2202751/193459480-7ef059d0-64bb-483c-94a7-f3d666446c3d.png)

**Comment by liborm85** [comment]
Fixed by PR https://github.com/bpampuch/pdfmake/pull/2764.
Released in version 0.2.11.
